packages/screens/Launchpad/LaunchpadCreateScreen.tsx

```diff
@@ -0,0 +1,220 @@
+import { LaunchpadDetails } from "./components/LaunchpadDetails";
+import { LaunchpadMinting } from "./components/LaunchpadMinting";
+import { LaunchpadTeamandInvestment } from "./components/LaunchpadTeamandInvestment";
+import ChevronRightSvg from "../../../assets/icons/chevron-right.svg";
+import { BrandText } from "../../components/BrandText";
+import { SVG } from "../../components/SVG";
+import { ScreenContainer } from "../../components/ScreenContainer";
+import { PrimaryBox } from "../../components/boxes/PrimaryBox";
+import { PrimaryButton } from "../../components/buttons/PrimaryButton";
+import { SecondaryButton } from "../../components/buttons/SecondaryButton";
+import { SpacerColumn } from "../../components/spacer";
+import { NetworkFeature } from "../../networks";
+import { ScreenFC, useAppNavigation } from "../../utils/navigation";
+import {
+  neutral17,
+  neutral22,
+  neutral33,
+  neutral77,
+  primaryColor,
+  primaryTextColor,
+} from "../../utils/style/colors";
+import { fontSemibold14 } from "../../utils/style/fonts";
+import { layout } from "../../utils/style/layout";
+
+const StepContent = ({ step }: { step: number }) => {
+  switch (step) {
+    case 1:
+      return <LaunchpadBasic />;
+    case 2:
+      return <LaunchpadDetails />;
+    case 3:
+      return <LaunchpadTeamandInvestment />;
+    case 4:
+      return <LaunchpadAdditional />;
+    case 5:
+      return <LaunchpadMinting />;
+    case 6:
+      return <LaunchpadAssetsandMetadata />;
+    default:
+      return <></>;
+  }
+};
+
+const stepOptions = [
+  { key: 1, title: "Basic" },
+  { key: 2, title: "Details" },
+  { key: 3, title: "Team & Investments" },
+  { key: 4, title: "Additional" },
+  { key: 5, title: "Minting" },
+  { key: 6, title: "Assets & Metadata" },
+];
+
+export const LaunchpadCreateScreen: ScreenFC<"LaunchpadCreate"> = () => {
+  const navigation = useAppNavigation();
+
+  const [selectedStep, setSelectedStep] = useState(1);
+  const [isLoading, setLoading] = useState(false);
+
+  const onSubmit = () => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 1000);
+  };
+
+  return (
+    <ScreenContainer
+      fullWidth
+      noMargin
+      noScroll={false}
+      footerChildren={<></>}
+      forceNetworkFeatures={[NetworkFeature.SocialFeed]}
+      headerChildren={<BrandText>Launchpad Submission Form</BrandText>}
+      onBackPress={() => navigation.navigate("LaunchpadApply")}
+    >
+      <View
+        style={{
+          marginTop: layout.spacing_x3,
+        }}
+      >
+        <PrimaryBox
+          style={{
+            flexDirection: "row",
+            alignItems: "center",
+            minHeight: 56,
+            minWidth: 1290,
+            backgroundColor: neutral17,
+            justifyContent: "center",
+            marginHorizontal: layout.spacing_x3,
+            borderWidth: 0,
+          }}
+        >
+          {stepOptions.map((item, index) => (
+            <TouchableOpacity
```

Comment:
Found this component too long, maybe you can extract this component ? `StepOptionItem` something like that